Makefile

```diff
@@ -10,6 +10,7 @@ PROTOC := $(shell which protoc)
 IMAGE_NAME = scorecard
 OUTPUT = output
 IGNORED_CI_TEST="E2E TEST:blob|E2E TEST:executable"
+PLATFORM="linux/amd64,linux/arm64,linux/386,linux/arm"
```

Comment:
What about `windows` and `darwin`?

Reply:
There is no docker containers for windows/darwin it is only for Linux.